reference/conanfile/attributes.rst

```diff
@@ -1348,6 +1348,32 @@ Its basic syntax is:
 
 Read more about this attribute in :ref:`python_requires`
 
+
+python_requires_extend
+----------------------
+
+.. warning::
+
+    This is an **experimental** feature subject to breaking changes in future releases.
+
+
+This class attribute defines one or more classes that will be injected in runtime as base classes of
```

Comment:
it would be nice to have an example of syntax of multiple classes specified in `python_requires_extend`, e.g. should I specify tuple, array, list or something else?

Reply:
Now the example uses several python requires.